example/l2fwd/README

```diff
@@ -5,3 +5,10 @@ Source code and Makefiles placed under test/performance/ directory.
 This L2 forwarding application can be used as example reference as well
 as performance test for different odp modes (direct, queue or scheduler
 with parallel, atomic or ordered queues).
+
+Note that this example is tuned for performance. As a result, when using
+ordered queues with direct packet I/O output or queued output with multiple
```

Comment:
I forgot in my previous comment that l2fwd support also direct pktin mode without the packet order problem. OTOH, in scheduled mode the issue affects atomic queues in addition to ordered queues.

More accurate wording would be something like this: "As a result, when using scheduled mode with direct packet I/O output or queued output with multiple output queues, packet order is not guaranteed."

Or maybe more simply: "As a result, packet order is not guaranteed when using scheduled mode."

Reply:
OK, used your suggested wording in v3.